elements/bulbs-video/plugins/google-analytics.js

```diff
@@ -0,0 +1,44 @@
+let prefixedSend = (gaPrefix) => {
+  return `${gaPrefix}.send`;
+};
+
+export default {
+  init (player, gaPrefix) {
+    let context = {
+      player: player,
+      gaPrefix: gaPrefix,
+    };
+    player.on('play', this.onPlay.bind(context));
```

Comment:
I'm not sure it makes sense to bind context as `this`.
Could drop the `context` structure entirely and bind `player` and `gaPrefix` as arguments.
```js
this.onPlay.bind(this, player, gaPrefix);
```

So onPlay changes to:

```js
onPlay (player, gaPrefix, event) {
  if (!player.playedOnce) {
/* and so on */
```